edx_management_commands/management_commands/management/commands/manage_group.py

```diff
@@ -0,0 +1,131 @@
+"""
```

Comment:
@clintonb the original name for this repo / project was edx-django-extensions.   over the course of review / feedback it's been renamed to edx-management-commands.  sorry for confusion.  i think i've updated everything necessary other than the name of the repo itself.